ui/prompt.go

```diff
@@ -54,24 +50,21 @@ func hasTeams(projects []*entity.Project) bool {
 
 func promptTeams(projects []*entity.Project) (*string, error) {
 	if hasTeams(projects) {
-		tm := make(map[string]bool)
+		teams := make([]string, 0)
+		teamCheck := make(map[string]bool)
 		for _, project := range projects {
-			if project.Team != nil {
-				tm[*project.Team] = true
+			if project.Team == nil {
+				continue
+			}
+
+			// Ensure teams are only appended once by checking teamCheck
+			if _, hasSeenTeam := teamCheck[*project.Team]; !hasSeenTeam {
+				teams = append(teams, *project.Team)
+				teamCheck[*project.Team] = true
```

Comment:
I'm surprised Go doesn't have an Object.values like way to convert map to list of values

Reply:
I am not at all surprised by this

https://twitter.com/JustJake/status/1395590075824541697

Reply:
Once Generics are in, this should be possible